docs/building-deployer-helm.md

```diff
@@ -623,69 +610,71 @@ general
 [onboarding guide](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/set-up-environment#product-identifiers).
 See [this section](building-deployer.md#publishing-a-new-version)
 for more information about tags. Let's say your application has a
-track named `v6`.
+track named `v6.0.0`.
 
 Your GCR repo should have the following images:
 
 ```text
-gcr.io/your-company/wordpress:v6
-gcr.io/your-company/wordpress/deployer:v6
-gcr.io/your-company/wordpress/mariadb:v6
+gcr.io/your-company/wordpress:v6.0.0
```

Comment:
if we change the primary image to give it an actual name `wordpress`, this will become `gcr.io/your-company/wordpress/wordpress:9.0.3`.

Reply:
Done!